tools/common/schema/setuptask.go

```diff
@@ -49,7 +51,7 @@ func newSetupSchemaTask(db DB, config *SetupConfig) *SetupTask {
 // Run executes the task
 func (task *SetupTask) Run() error {
 	config := task.config
-	log.Printf("Starting schema setup, config=%+v\n", config)
+	config.Logger.Info("Starting schema setup", tag.NewAnyTag("config", config))
```

Comment:
nit: would be clear to have a local variable for logger := config.Logger so you can do logger.Info() instead of config.Logger.Info().